schemas/responses/userTasksState.yaml

```diff
@@ -0,0 +1,37 @@
+UserTasksState:
+  type: object
+  required:
+    - userTasks
+    - version
+  properties:
+    userTasks:
+      type: array
+      items:
+        type: object
+        required:
+          - UserTaskId
+          - RequestURL
+          - ClientIdentity
+          - StartMs
+          - Status
+        properties:
+          UserTaskId:
+            type: string
+          RequestURL:
+            type: string
+          ClientIdentity:
+            type: string
+          StartMs:
+            type: string
+          Status:
+            type: string
+            enum:
+              - Active
+              - InExecution
+              - Completed
+              - CompletedWithError
+          originalResponse:
+            type: string
```

Comment:
In the future we will have to move this into its own component/class, because of the way responses are tested. Up to you if you want to move it to a new yaml right now or wait till later.

Reply:
Does every object needs to be in a separate class?